types/configurable/configurable-tests.ts

```diff
@@ -0,0 +1,100 @@
+import configureFunction, { Configurable } from 'configurable';
```

Comment:
the runtime node module does nto have a name `Configurable` on it. i do not think this is correct.

Reply:
Removed.